src/components/CohortBuilder/Summary/FileBreakdown.js

```diff
@@ -4,24 +4,13 @@ import { withTheme } from 'emotion-theming';
 import { compose } from 'recompose';
 import { CohortCard } from './ui';
 import BaseDataTable from 'uikit/DataTable';
+import MultiHeader from 'uikit/Multicard/MultiHeader';
 
-const FileBreakdownWrapper = styled('div')`
-  margin-top: 10px;
-`;
+const FileBreakdownWrapper = styled('div')``;
 
-const TableFooter = styled('div')`
-  text-align: right;
-  padding-top: 13px;
-  font-size: 11px;
-  font-weight: bold;
-  color: ${({ theme }) => theme.secondary};
-  & a {
-    font-weight: normal;
-    color: ${({ theme }) => theme.hover};
-    text-decoration: underline;
-    margin-left: 5px;
-  }
-`;
+const columnStyles = {
+  margin: '-2% 0',
```

Comment:
@ciaranschutte does `BaseDataTable` expose the full `react-table` API? if so, there might be better ways to apply styles to the rows. Regardless, `-2%` margin seems odd to me as it will not be consistent across viewport size, can we try a fixed `px` value? :)

Reply:
Using % is more responsive than px

Reply:
I don't think we want it to respond to the width, it looks odd in a certain screen size, easier to just fix the `px` if possible
![screen shot 2019-03-05 at 2 25 35 pm](https://user-images.githubusercontent.com/11821106/53831427-ccf53180-3f52-11e9-90e8-404f86856782.png)

Reply:
@hlminh2000 not whole API but most.
`style` passed in with `Column` is valid in their API ... I don't like the feel of it either

Reply:
ok that's fine then :) 
anyway, `-5px 0` seems to work well for me here